### PR TITLE
DOC: Fix camelCase (or capitalization)

### PR DESCRIPTION
**Pull request by ayushman17**
Docs have been updated (for bug fixes/features)

docs update => proper capitalisation would be better. 👍: Github

## PR Checklist
* [x] CLA signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/Terminal) and sign the CLA

### Other information:
Signed-off-by: Ayushman Singh Chauhan <ascb508@gmail.com>

### CODE REVIEW

**Comment by DHowett** [comment]
Sorry for the delay in reviewing this! Much of the team is unavailable this week, so we're short-staffed. We'll get to the backlog of issues and PRs as soon as possible.

File: doc/specs/#1043 - Set the initial position of the Terminal/spec.md [hunk 1]
```diff
@@ -9,15 +9,15 @@ issue id: #1043
 
 ## Abstract
 
-This spec is for task #1043 �Be able to set an initial position for the terminal�. It goes over the details of a new feature that allows users to set the initial position and size of the terminal. Expected behavior and design of this feature is included. Besides, future possible follow-up works are also addressed.
+This spec is for task #1043 “Be able to set an initial position for the terminal”. It goes over the details of a new feature that allows users to set the initial position and size of the terminal. Expected behavior and design of this feature is included. Besides, future possible follow-up works are also addressed.
```
Comment (lhecker) [inline doc/specs/#1043 - Set the initial position of the Terminal/spec.md:12]: If anyone wonders what's different here: The old/current file is encoded as CP 1252 and this PR encodes it as UTF-8.
In my opinion you can feel absolutely free to just replace these uncommon Unicode quotes and dashes with standard ASCII ones. None of the other doc files uses those anyways.

**Review by lhecker — APPROVED** [review]

File: doc/specs/#1043 - Set the initial position of the Terminal/spec.md (hunk 1 above)
Comment (ayushman17) [reply doc/specs/#1043 - Set the initial position of the Terminal/spec.md:12]: screen
yeah, I can see them here...

**Comment by carlos-zamora** [comment]
Thanks @ayushman17 !

**Comment by ayushman17** [comment]
I THANK YOU for the creation of this tool! 
and look forward to continued open-source contribution!

**Pull request merged**